src/index.ts

```diff
@@ -105,7 +105,8 @@ export async function createServer() {
     host: COUNTRY_CONFIG_HOST,
     port: COUNTRY_CONFIG_PORT,
     routes: {
-      cors: { origin: whitelist }
+      cors: { origin: whitelist },
+      payload: {maxBytes: 52428800}
```

Comment:
Could this also be a reference to the `ACCUMULATED_FILE_SIZE` variable?

Reply:
@rikukissa lets leave this at 50MB.  We can have the ACCUMULATED_FILE_SIZE as an optional between 20MB and 50MB limit.  But I am not sure if we should persist with ACCUMULATED_FILE_SIZE.  See my comment here: https://github.com/opencrvs/opencrvs-core/pull/3257#discussion_r884990961